### PR TITLE
Got the services up and running 

### PR DESCRIPTION
**Pull request by jesper-friis**
- use a branch in oteapi-core with a quickfix that skips await in datacache
- improve Dockerfile:
  - only copy what is needed into the docker image
  - added some commented out code that installs oteapi-core such that it can be debugged while the docker is running
- removed some old files

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -31,16 +31,27 @@ RUN mkdir -p /app/entities
...
-RUN pre-commit run --all-files  \
-  && safety check -r requirements.txt -r requirements_dev.txt
+#RUN pre-commit run --all-files  \
+#  && safety check -r requirements.txt -r requirements_dev.txt
 
 # Run pytest with code coverage
 # RUN pytest --cov app
 
+# For debugging: install oteapi locally
+#COPY oteapi-core oteapi-core
```
Comment (francescalb) [inline Dockerfile:48]: The local oteapi-core should not be inside oetapi-services. Copy from path to is my suggstion.
```suggestion
# copy oteapi-core from local path
#COPY ../oteapi-core oteapi-core
```
Furthermore, this does not allow for runing services locally while developing oteapi-core. That would require  mounting the oteapi-core as a volume and pip installing from there. 

I have tried to do that but right now the services fail as soon as there is an error, making everything stop.

File: Dockerfile [hunk 2]
```diff
@@ -31,16 +31,27 @@ RUN mkdir -p /app/entities
...
+#RUN pip install -U /app/oteapi-core
```
Comment (francescalb) [inline Dockerfile:49]: Installing for development of oteapi-core needs pip install -e . on volume mounted.

File: Dockerfile (hunk 1 above)
Comment (jesper-friis) [reply Dockerfile:55]: Agree, placing oteapi-core inside oteapi-services is not the correct way to do things. It is just a hack that makes it much easier to debug issues within oteapi-core that makes the services crash.

File: Dockerfile [hunk 3]
```diff
@@ -31,16 +31,34 @@ RUN mkdir -p /app/entities
...
 RUN pip install -q --trusted-host pypi.org --trusted-host files.pythonhosted.org -r requirements_dev.txt
-RUN pre-commit run --all-files  \
-  && safety check -r requirements.txt -r requirements_dev.txt
+
+COPY wsgi.py ./
+COPY app app
+
+# TODO: move these to a plugin for the compevo case
+COPY entities entities
+COPY python_mapping_plugins python_mapping_plugins
```
Comment (CasperWA) [inline Dockerfile:44]: I guess these shouldn't be included?

File: requirements.txt [hunk 4]
```diff
@@ -29,7 +29,7 @@ invoke==1.6.0
 kombu==5.2.3
 numpy==1.22.1
 openpyxl==3.0.9
-oteapi-core @ git+https://github.com/EMMC-ASBL/oteapi-core.git@4a7f087e464d9ac017aeb5e5a96be659e4816176
+oteapi-core @ git+https://github.com/EMMC-ASBL/oteapi-core.git@90961930b7c1f83466c28a7e03dc04122a98e812
```
Comment (CasperWA) [inline requirements.txt:32]: Couldn't we point to `master` instead?

**Review by CasperWA — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 4 above)
Comment (jesper-friis) [reply requirements.txt]: Good point!

**Review by CasperWA — APPROVED** [review]
Cool - let's get it in

**Pull request merged**